src/main/java/org/openrewrite/maven/AbstractRewriteBaseRunMojo.java

```diff
@@ -55,6 +59,10 @@ public abstract class AbstractRewriteBaseRunMojo extends AbstractRewriteMojo {
     @Parameter(property = "rewrite.exportDatatables", defaultValue = "false")
     protected boolean exportDatatables;
 
+    @Parameter(property = "rewrite.options")
+    @Nullable
+    protected String options;
```

Comment:
One thing left to try here is if this could just be a list to remove one of the split operations below

Reply:
Not for when using command line ARGS, maven don't parse complex types

Reply:
Hmm; I thought we had that working for these two options by using LinkedHashSet
https://github.com/openrewrite/rewrite-maven-plugin/blob/1d2d8aa4ac246e7b0d47d3b4762c96379cb7ee92/src/main/java/org/openrewrite/maven/ConfigurableRewriteMojo.java#L54-L60

My tests are hampered by Sonatype OSS unavailability, so I can't test right now, but I'd like to give that a shot before a merge.